lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -26,10 +26,12 @@
                  role="columnheader"
                  tabindex="0"
                  [attr.aria-sort]="col.sortable ?
-                    ((isColumnSorted(col, 'asc') ?
-                        'ADF-DATATABLE.ACCESSIBILITY.SORT_ASCENDING':
-                        'ADF-DATATABLE.ACCESSIBILITY.SORT_DESCENDING') | translate) :
-                     null"
+                    ((isColumnSortActive(col) ?
```

Comment:
seems to be too much code responsibility for the html template. could you move it to a single typescript function and just trigger it from the attribute? like `getAriaSort(col)` or similar